src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -62,29 +62,29 @@ const styles = StyleSheet.create({
     fontSize: 10,
     lineHeight: 12.19,
     alignSelf: 'center'
+  },
+  mr8: {
+    marginRight: 8
   }
 });
 
 const ChatItemTargetText = ({
-  avatar = DEFAULT_PROFILE_PIC_PATH,
+  avatar,
   username = 'Anonymous Clown',
   time = '4h',
   isContinuous = false,
-  message = 'Ultrices neque op semper blahbla blahri mauris amet, penatibus. pi Amet, mollis quam venenatis di'
+  message = 'Ultrices neque op semper blahbla blahri mauris amet, penatibus. pi Amet, mollis quam venenatis di',
+  AnonymousImage
 }: ChatItemMyTextProps) => {
+  const renderAvatar = React.useCallback(() => {
+    if (isContinuous) return <View style={styles.avatar} />;
+    if (AnonymousImage) return <View style={styles.mr8}>{AnonymousImage}</View>;
+    return <FastImage style={styles.avatar} source={{uri: avatar ?? DEFAULT_PROFILE_PIC_PATH}} />;
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ChatItemTargetText` component has been updated to include a new prop `AnonymousImage`. This prop is used in the `renderAvatar` function. However, there's no type definition for `AnonymousImage` in `ChatItemMyTextProps`. It's important to update the type definitions to ensure type safety and better developer experience.

```diff
-  ChatItemMyTextProps
+  ChatItemMyTextProps & { AnonymousImage?: React.ReactNode }
```

Also, the `useCallback` hook is used without any dependencies which means it will create a new function on every render. If there are no dependencies that would cause this callback to change, you can remove the `useCallback` entirely for better performance.

```diff
- const renderAvatar = React.useCallback(() => {
+ const renderAvatar = () => {
```

Finally, the `avatar` prop now defaults to `undefined` instead of `DEFAULT_PROFILE_PIC_PATH`. This could potentially break existing usage where the default avatar path was expected. If this change is intentional and all usages have been updated accordingly, then it's fine. Otherwise, consider reverting this change.

```diff
- avatar,
+ avatar = DEFAULT_PROFILE_PIC_PATH,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->